hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -279,6 +279,7 @@ protected OmMetadataManagerImpl() {
 
   @Override
   public Table<String, OmKeyInfo> getKeyTable() {
+    // TODO: Refactor the below function by reading bucketLayout.
```

Comment:
Please raise a jira task to address this TODO for tracking it. Also, please mention the jira id here in TODO section.

Reply:
Done